op/k8s/apiserver_restart.go

```diff
@@ -311,6 +315,13 @@ func APIServerParams(controlPlanes []*cke.Node, advertiseAddress, serviceSubnet
 				Propagation: "",
 				Label:       cke.LabelShared,
 			},
+			{
+				Source:      "/var/log/audit",
```

Comment:
This should not be embedded in CKE.
Instead, users should mount a host directory by `extra_binds`.